js/hw1.js

```diff
@@ -0,0 +1,10 @@
+const nameUser = function(name) {
+    if (name === 'Mark') {
```

Comment:
What if I pass lower cased word 'mark'. Seems that code will not work as expected. Please, add logic to ignore case sensitive issue